docs/refman/loaderyaml.py

```diff
@@ -306,3 +306,8 @@ def load_impl(self) -> ReferenceManual:
                     [self._load_module(x) for x in self.modules_dir.iterdir()]
                 ], flatten=True)
             )
+
+            if not self.strict:
+                mlog.warning('YAML reference manual loaded using the best-effort fastyaml loader.  Results are not guaranteed to be stable or correct.')
+
+            return manual
```

Comment:
Hmm, why isn't the mlog.warning just moved up a drop?

Reply:
Oh... maybe this is actually set by the `ReferenceManual()` invocation?

Reply:
Are you saying the `if not self.strict` should be unnecessary?

Reply:
No, I just don't understand why you assign `manual`, *then* do `if not self.strict: mlog.warning`, then return `manual`.

Just do `if not self.strict: mlog.warning` and then `return ReferenceManual(...)`.

Reply:
Ah, I see. Makes sense.